### PR TITLE
Add flag `--yes` from `create` task (#33)

### PR DESCRIPTION
**Pull request by kostyabushuev**

### CODE REVIEW

**Comment by nikolay-govorov** [comment]
Fixed #33

**Comment by nikolay-govorov** [comment]
Hey. Thank you for PR. I wrote [commen](https://github.com/shower/cli/issues/33#issuecomment-489421984) about why the flag should be called `--yes` and not `--all`. Let's rename it?

**Comment by nikolay-govorov** [comment]
Also, now you've added a parameter, just a positional string, and you need a flag. It can be added using the `option` method instead of the `positional` method

**Comment by kostyabushuev** [comment]
@nikolay-govorov 
1. change to '--all'
2. use option `instead` of `positional`

update commit

File: source/commands.js [hunk 1]
```diff
@@ -37,12 +37,17 @@ async function applyCommand (name, env, options) {
 const list = {
   create: {
     aliases: ['new'],
-    meta: '[<directory>]',
+    meta: '[<directory>] [yes|y]',
```
Comment (nikolay-govorov) [inline source/commands.js]: Since this is a flag, we do not include it in the task description. You can get it on `--help`.

Comment (kostyabushuev) [reply source/commands.js]: sorry for this mistakes
thanks for your help

updated commit

**Comment by nikolay-govorov** [comment]
Thanks for the help, merge this :D

**Pull request merged**